README.md

```diff
@@ -207,15 +207,25 @@ For more information about the navigation bar, click [here](https://docusaurus.i
 
 For more information about custom pages, click [here](https://docusaurus.io/docs/en/custom-pages).
 
-# Updating the API Reference
+# Updating the API Reference & Storefront Reference
 
-All files in /docs/api-reference are generated by @graphql-markdown/docusaurus package. Introduction page is automatically copied from `/docs/api-introduction.mdx` file.
+Files in /docs/api-reference and /docs/api-storefront are generated by `@graphql-markdown/docusaurus` package. Introduction pages are automatically copied from `/template/api-reference.mdx` and `/template/storefromt/api-reference.mdx` files.
+
+## Updating
 
 To update the API reference:
 
-1. Start Saleor API at `http://localhost:8000`
+1. Put `schema.graphql` locally into the root of the saleor-docs repository
 1. Run `npm run update-api-reference`
 
+The script behind the scenes does several steps:
+
+1. The api-reference is generated in `.tmp` folder.
+1. The api-storefront is generated in `.tmp` folder. This step also includes the generation of storefront schema. Storefront schema is a subset of the Saleor schema with filtered-out elements that require permissions.
```

Comment:
> with filtered-out elements that require permissions.

I know this is not the scope of this PR, but IMO we need to change this approach, and not include mutations/queries that require OWNER or AUTHENTICATED_USER. Currently, any mutation like set-default-shipping-address for customer, are placed in api-reference, instead of a storefront-api